docs/DeveloperGuide.adoc

```diff
@@ -839,6 +907,18 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
+|`* * *` |user |add an event |
```

Comment:
As I mentioned in the 'Member' section, you do not need to list down all the possible commands here. It is supposed to be available in the user guide. Perhaps, you may illustrate a sequence of actions here similar to the undo/redo section.